source/Ship.h

```diff
@@ -305,6 +305,9 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	void AddCrew(int count);
 	// Check if this is a ship that can be used as a flagship.
 	bool CanBeFlagship() const;
+	// Get and set the number of desired crew members when this is a flagship.
```

Comment:
We might want to use this for non-flagships as well in the future (if defense from capturing becomes relevant again).

```suggestion
	// Desired crew members for this ship. Having more than the minimum crew increases the odds when capturing and when defending from capture. (Currently only used for flagships.)
```